dask/config.py

```diff
@@ -0,0 +1,194 @@
+from __future__ import print_function, division, absolute_import
+
+from contextlib import contextmanager
+import os
+import sys
+
+from .compatibility import FileExistsError
+
+
+config_paths = [
```

Comment:
These appear windows-unfriendly

Reply:
Yeah, I'm not yet sure what the windows equivalent is for `/etc/`.  This is still a TODO.  Wisdom welcome here.